### PR TITLE
negar/tnc_forex_launch

### PR DESCRIPTION
**Pull request by negar-binary**
update forex tnc

### CODE REVIEW

File: src/templates/static/tac.html.tt [hunk 1]
```diff
@@ -138,6 +141,8 @@
...
+                            <li>[% l("Our Prices are determined by reference to the price of the underlying instrument which is quoted on external exchanges or dealing facilities that we select at our discretion. Our IT infrastructure and price aggregator system facilitates the reception of quotes from shortlisted prime brokers who act as liquidity providers for us and the delivery of the quotes, derived with reference against relevant benchmark and markets, to you. You acknowledge that our Prices may differ from the bid and offer prices made available by external exchanges or dealing facilities due to network latency and we are not liable for any losses that you incur arising from such differences. Refer to our Product Disclosure Statement for our pricing methodology and factors. This clause does not apply to Volatility Indices.") %]</li>
```
Comment (jy-deriv) [inline src/templates/static/tac.html.tt]: remove the words "due to network latency" from the sentence

Comment (negar-binary) [reply src/templates/static/tac.html.tt]: Done

**Comment by jy-deriv** [comment]
pls check conflicts

**Comment by negar-binary** [comment]
@jy-binary done

**Pull request merged**